influx-writer/requirements.txt

```diff
@@ -1,5 +0,0 @@
-ujson==1.33
```

Comment:
Just checking that the main `requirements` file contains all of the items from these deleted files (my guess is yes, since you've got it running)